release/llm_tests/serve/run_llm_serve_release_tests.py

```diff
@@ -94,6 +119,85 @@ def main(
 
             logger.info(f"Performance test results: {results}")
 
+            accelerator = llm_config.get("accelerator_type", "NoGpu")
+            # "A10" without "G" to match existing dashboard tags
+            if accelerator == "A10G":
+                accelerator = "A10"
+
+            tensor_parallel_size = llm_config["engine_kwargs"].get(
+                "tensor_parallel_size", 0 if accelerator == "NoGpu" else 1
+            )
+            tag = f"{accelerator}-TP{tensor_parallel_size}"
+            for result in results:
+                record = FirehoseRecord(
+                    record_name=RecordName.RAYLLM_PERF_TEST,
+                    record_metrics={
+                        "api_url": api_url,
+                        "api_token": api_token,
+                        "cloud_name": CLOUD,
+                        "service_name": SERVICE_NAME,
+                        "py_version": get_python_version_from_image(image_uri),
+                        "tag": tag,
+                        "vllm_engine": f"V{VLLM_USE_V1}",
+                        **result,
+                    },
+                )
+                record.write(verbose=True)
+
+    if run_perf_profiler:
+        anyscale.job.wait(
+            id=submitted_job_id,
+            state=anyscale.job.JobState.SUCCEEDED,
+            timeout_s=JOB_TIMEOUT_S,
+        )
+
+        # Read data from bucket and send to anyscale-dev-product's Firehose
+        # This Firehose is where databricks has access to.
+        data_for_firehose = read_from_s3(s3_storage_path)
+
+        for result in data_for_firehose:
+            record = FirehoseRecord(
+                record_name=RecordName.VLLM_PERF_TEST,
+                record_metrics=result,
+            )
+            record.write(verbose=True)
+
+
+def submit_benchmark_vllm_job(image_uri: str, serve_config_file: str, hf_token: str):
+    py_version = get_python_version_from_image(image_uri)
```

Comment:
Is this python version gonna be ever different than the python version that is running this script? Can we simply use a local information to not pass py_version around?